### ISSUE TITLE
Is there way to extract gamma Tensor in BN layer?

### ISSUE THREAD

**Issue by rimchang** [Cadene/tensorflow-model-zoo.torch#5]
why not store gamma tensor?
Is there way to extract gamma Tensor?
I can't find gamma tensor in tensorflow graph

https://github.com/Cadene/tensorflow-model-zoo.torch/blob/master/inceptionresnetv2/tensorflow_dump.py
  beta = sess.graph.get_tensor_by_name('InceptionV4/'+name+'/BatchNorm/beta:0').eval()
  #gamma = sess.graph.get_tensor_by_name('InceptionV4/'+name+'/BatchNorm/gamma:0').eval()
  mean = sess.graph.get_tensor_by_name('InceptionV4/'+name+'/BatchNorm/moving_mean:0').eval()
  var = sess.graph.get_tensor_by_name('InceptionV4/'+name+'/BatchNorm/moving_variance:0').eval()

**Comment by Cadene** [comment]
As far as I remember, sometimes gamma tensor is not used in the tensorflow implementation.

However in Nasnet, it is used:

- https://github.com/Cadene/tensorflow-model-zoo.torch/blob/master/nasnet/tensorflow_dump.py#L155
- https://github.com/Cadene/tensorflow-model-zoo.torch/blob/master/nasnet/pytorch_load.py#L40

Did you encounter a problem with InceptionResnetV2?
